pkg/operator/client/client.go

```diff
@@ -72,7 +72,11 @@ func GetScheme() *runtime.Scheme {
 
 // NewClient builds an operator-compatible kube client from the given REST config.
 func NewClient(kubeConfig *rest.Config) (client.Client, error) {
-	mapper, err := apiutil.NewDiscoveryRESTMapper(kubeConfig)
+	httpClient, err := rest.HTTPClientFor(kubeConfig)
+	if err != nil {
+		fmt.Errorf("failed to create HTTPClient: %v", err)
```

Comment:
```suggestion
		return nil, fmt.Errorf("failed to create http client: %v", err)
```
I changed the error message just to be consistent with the other errors from this function.

Reply:
Whoops good catch.